tools/loader/ImageClassifier.cpp

```diff
@@ -319,12 +327,34 @@ static void processAndPrintResults(Tensor *SMT, llvm::StringRef functionName) {
   }
 }
 
+/// Read all images from \p inputImageListFile in to \p inputImageFilenames
+static void parseInputImageList(const std::string &inputImageListFile) {
+  std::ifstream inFile;
+  inFile.open(inputImageListFile);
+  while (!inFile.eof()) {
+    std::string img;
+    getline(inFile, img);
+    if (!img.empty()) {
+      inputImageFilenames.push_back(img);
+    }
+  }
+  inFile.close();
+}
+
 int main(int argc, char **argv) {
   PlaceholderBindings bindings;
   // The loader verifies/initializes command line parameters, and initializes
   // the ExecutionEngine and Function.
   Loader loader(argc, argv);
 
```

Comment:
Can you add a conditional here checking if `inputImageFilenames.size() == 0 && inputImageListFile.empty()`, and then print to `llvm::errs()` with an explanation that one of them must be used, and then `std::exit(1);`. We do something like this in `Loader.cpp` with `commandLineIsInvalid()`.

Reply:
Sure, makes sense to add.